controllers/drpolicy_controller.go

```diff
@@ -113,6 +113,11 @@ func (r *DRPolicyReconciler) Reconcile(ctx context.Context, req ctrl.Request) (c
 		return ctrl.Result{}, err
 	}
 
+	if utils.IsStorageClientType(mirrorPeer.Spec.Items) {
+		logger.Info("MirrorPeer contains StorageClient reference. Skipping creation of VolumeReplicationClasses", "MirrorPeer", mirrorPeer.Name)
```

Comment:
```suggestion
		logger.Info("MirrorPeer contains StorageClient reference. Skipping creation of VolumeReplicationClasses")
```
Isn't MirrorPeer name already injected in the main reconcile function?

Reply:
This is DRPolicy controller, only the req is injected with the logger. MirrorPeer is fetched separately